bot.py

```diff
@@ -65,11 +65,15 @@ def __init__(self, *args, **kwargs) -> None:
         self.session = aiohttp.ClientSession(loop=self.loop)
         self.load_extension("jishaku")
         self.loop.run_until_complete(self.create_db_pool())
+        self.mongo = motor.motor_asyncio.AsyncIOMotorClient(str(self.connection_url))
+        self.mongodb = self.mongo["pybot"]
+        self.welcomer = Document(self.mongodb, "welcomer")
```

Comment:
Since the `on_ready` triggers more than once. It would be better to place them in `__init__`